tracking/translations_parser/parser.py

```diff
@@ -36,6 +40,10 @@
 # Expected version of Marian for a clean parsing
 SUPPORTED_MARIAN_VERSIONS = [(1, 10), (1, 12)]
 
+MARIAN_ARGS_REGEX = re.compile(r"command line:[\n ]+[\w\/-]+\/marian +(.*)")
+# Last Marian command line argument (not being part of training extra arguments)
+LAST_MARIAN_DECLARED_ARGUMENT = "seed"
```

Comment:
this is fragile and will change after the refactoring (rewriting the train.sh in python)

Reply:
So, I don't think it's possible to distinguish what args are additional from inside the script. We can just log all the marian args then. Having all of them also can be useful.